### ISSUE TITLE
some Latex based formulas not shown on Kiwix Android APK 3.3.0/3.2.1

### ISSUE THREAD

**Issue by florian29** [kiwix/kiwix-android#2032]
**Describe the bug**
Some Latex based formulas are not shown on Kiwix Android APK 3.3.0 and 3.2.1 when opening articles with many formulas with the ZIM file "wikipedia_de_all_maxi_2020-04.zim". You must have opened at least 3 big articles with many formulas.

**Expected behavior**
All the formulas should be shown despite of how many articles you opened before. The internal data base should be cleared by the app by itself to ensure that every article is shown correctly.

**Steps to reproduce the behavior:**
Open the ZIM file "wikipedia_de_all_maxi_2020-04.zim", then open some articles with many formulas, like "Maxwell-Gleichungen" and "Satz des Pythagoras". Scroll down the whole articles, so that the whole articles with all of the formulas have been shown. Then open "Sinus und Kosinus" and you will see the article with symbols of broken images:

![Screenshot_2020-04-24-14-47-26](https://user-images.githubusercontent.com/43439672/80229802-3a7d8900-8651-11ea-863c-7f23f582d432.png)

Than go to the Android Application Manager and clear the data (clearing the cache is not sufficient!) and force stopping the Kiwix App. Than reopen the Kiwix App. Open again the article "Sinus and Kosinus" and than everything seems to be normal:

![Screenshot_2020-04-24-15-00-14](https://user-images.githubusercontent.com/43439672/80229923-731d6280-8651-11ea-9c18-e3a1d8711906.png)

Maybe it's a problem with overloading the internal data cache of the App or something like this. It should be a problem which can be solved. You should care about it, because it's not very comfortable to do this workaround (deleting the data of the app) after opening an article.
Thank you in advance

**Environment**
- Version of Kiwix Android : 3.3.0 and 3.2.1
- Device : Samsung Galaxy Tab SM-T530 and also with Samsung Galaxy S4 mini
- OS version : Android 5.0.2 and also 4.4.4

**Comment by kelson42** [comment]
I can not reproduce the problem (tested with a smaller ZIM file) on a recent Android.

**Comment by macgills** [comment]
It will take me an hour or two to download the zim but I am testing

**Comment by kelson42** [comment]
@macgills Considering that the android is old, device should be relatively slow. Do we have a kind of timeout in content loading?

**Comment by florian29** [comment]
I tried to reproduce again the bug which I described, but this time it was necessary to open a lot more articles then I described above to get the issue. At least I could get one broken image in the article "Tangens" after opening about 8 articles one by one. This time clearing the cache solved the problem. By the way: Despite the device is old, the Kiwix app and the zim file are opening very quickly. Opening a huge article just needs a few seconds. Maybe it would be a good idea to add an option in the configuration to empty the cache automatically after leaving the app? For those users which have this problem.
